engine/runtime/src/main/java/org/enso/interpreter/runtime/util/TruffleFileConversions.java

```diff
@@ -0,0 +1,61 @@
+package org.enso.interpreter.runtime.util;
+
+import com.oracle.truffle.api.TruffleFile;
+import java.io.File;
+import org.enso.filesystem.FileSystem;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.pkg.Package;
+
+/**
+ * A collection of utilities for converting between various structures containing {@link TruffleFile} to
+ * their {@link File} counterparts.
+ *
+ * This is needed when we pass data from the {@code runtime} project into {@code runtime-compiler},
+ * as {@code runtime-compiler} does not depend on truffle API.
+ */
+public class TruffleFileConversions {
+  private TruffleFileConversions() {}
+
+  public static Package<File> convertTruffleFilePackage(Package<TruffleFile> trufflePkg) {
```

Comment:
Wild conversion. Does the `runtime/compiler` need the `File` from the `Package` at all?

Reply:
Will be addressed in another issue - #8888